lib/services/northBound/contextServer.js

```diff
@@ -733,44 +1080,70 @@ function updateErrorHandling(error, req, res, next) {
  */
 function loadContextRoutes(router) {
     //TODO: remove '//' paths when the appropriate patch comes to Orion
-    var updateMiddlewares = [
+    var updateMiddlewaresNgsi1 = [
+            middlewares.ensureType,
+            middlewares.validateJson(updateContextTemplateNgsi1),
+            handleUpdateNgsi1,
+            updateErrorHandling
+        ],
+        updateMiddlewaresNgsi2 = [
             middlewares.ensureType,
-            middlewares.validateJson(updateContextTemplate),
-            handleUpdate,
+            middlewares.validateJson(updateContextTemplateNgsi2),
+            handleUpdateNgsi2,
             updateErrorHandling
         ],
-        queryMiddlewares = [
+        queryMiddlewaresNgsi1 = [
             middlewares.ensureType,
             middlewares.validateJson(queryContextTemplate),
-            handleQuery,
+            handleQueryNgsi1,
+            queryErrorHandling
+        ],
+        queryMiddlewaresNgsi2 = [
+            handleQueryNgsi2,
             queryErrorHandling
         ],
-        updatePaths = [
+        updatePathsNgsi1 = [
             '/v1/updateContext',
             '/NGSI10/updateContext',
             '//updateContext'
         ],
-        queryPaths = [
+        updatePathsNgsi2 = [
+            '/v2/op/update'
+        ],
+        queryPathsNgsi1 = [
             '/v1/queryContext',
             '/NGSI10/queryContext',
             '//queryContext'
+        ],
+        queryPathsNgis2 = [
+            '/v2/entities',
+            '/v2/entities/:entity',
```

Comment:
From https://github.com/telefonicaid/fiware-orion/issues/3068#issuecomment-468584414 only the fist one (`'/v2/entities'`) should be taken into acocunt. I'd suggest to remove the other (as much, leave a comment here explaining that an evolved implementation could add more ones).

Focusing on `GET /v2/entities` from [NGSIv2 spec](http://fiware.github.io/specifications/ngsiv2/stable) note that `GET /v2/entities` may have 15 different parameters but not all them are relevant in the context of query forwardning. I'd suggest to limit to:

* id
* type

Not sure about `idPattern`, `typePattern` and `q`... we cannot be sure if they are going to be used (it depends on how https://github.com/telefonicaid/fiware-orion/issues/3068 gets implemented in Orion at the end) but by the moment I think they can be leave out (maybe a reminder mark in the form of FIXME comment could be useful at some point).

Reply:
"attrs" query param should be also taken into account

Reply:
(To make it clearer I have added this to the "specification" in https://github.com/telefonicaid/fiware-orion/issues/3068#issuecomment-470005604)

Reply:
I agree, let's start with a minimum but functional implementation on both sides and then more functionalities can be added if needed.

Reply:
Fixed with 1ad3afb50c6a474150d68e5154ca4effa688948e